storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.12.rst

```diff
@@ -0,0 +1,33 @@
+===========================
+|Percona XtraBackup| 2.4.12
+===========================
+
+Percona is glad to announce the release of |Percona XtraBackup| 2.4.12 on
+June 21, 2018. Downloads are available from our `download site
+<http://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-2.4.12/>`_ and
+from :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+All Percona software is open-source and free.
+
+Bugs fixed
+==========
+
+* ``xtrabackup --copy-back`` didn't read plugins folder location from  ``plugin-dir`` setting of the ``my.cnf`` configuration file. Bug fixed :jirabug:`PXB-1544`.
+
+* xbstream was exiting with zero return code when it failed to create one or
+  more target files instead of returning error code 1. Bug fixed
+  :jirabug:`PXB-1542`.
+
+* Meeting a zero sized keyring file, |Percona XtraBackup| was removing and
+  immediately recreating it, which could affect external software noticing this
+  file had undergo manipulations. Bug fixed  :jirabug:`PXB-1540`.
+
+* |Percona XtraBackup| didn't print used arguments to standard output. Bug
```

Comment:
It was not a "bug", but rather an improvement. Can we say "now prints" instead of "did not print"?